resources/metadata/imagePushToRegistry.yaml

```diff
@@ -0,0 +1,171 @@
+metadata:
+  name: imagePushToRegistry
+  description: Allows you to copy a Docker image from a source container registry  to a destination container registry.
+  longDescription: |-
+    In case you want to pull an existing image from a remote container registry, a source image and source registry needs to be specified.<br />
+    This makes it possible to move an image from one registry to another.
+
+    The imagePushToRegistry is not similar in functionality to containerPushToRegistry (which is currently a groovy based step and only be used in jenkins).
+    Currently the imagePushToRegistry only supports copying a local image or image from source remote registry to destination registry.
+
+spec:
+  inputs:
+    secrets:
+      - name: dockerConfigJsonCredentialsId
+        description: Jenkins 'Secret file' credentials ID containing Docker config.json (with registry credential(s)).
+        type: jenkins
+      - name: targetRepositoryCredentialsId
+        description: Jenkins 'Username Password' credentials ID containing username and password for the Helm Repository authentication
+        type: jenkins
```

Comment:
Should we support Jenkins credentials, or should we only fetch credentials from Vault?

Reply:
i think we should encourage vault otherwise part of the credentials are in Jenkins and part in vault, it will be confusing. for a starter lets avoid it